python/rmm/rmm/pylibrmm/cuda_stream_pool.pyi

```diff
@@ -0,0 +1,18 @@
+# SPDX-FileCopyrightText: Copyright (c) 2021-2025, NVIDIA CORPORATION.
+# SPDX-License-Identifier: Apache-2.0
+
+from enum import IntEnum
+from typing import Optional, cast
+
+from rmm.pylibrmm.stream import Stream
+
+class CudaStreamFlags(IntEnum):
+    SYNC_DEFAULT = cast(int, ...)
+    NON_BLOCKING = cast(int, ...)
+
+class CudaStreamPool:
+    def __init__(
+        self, pool_size: int = 16, flags: CudaStreamFlags = CudaStreamFlags.SYNC_DEFAULT
```

Comment:
Should some of these “int” parameters and return values be size_t instead? Even if Python treats them as Python int, I believe Cython will do some validation for signedness and size when converting arbitrary Python ints.